src/scripts/git_data.py

```diff
@@ -13,6 +17,36 @@ def get_last_commit_date(file_path):
         print(f"An error occurred: {e}")
     return None
 
+def get_last_commit_date(file_path):
+    try:
+        # get the last commit date as "September 12, 2022"
+        GITHUB_TOKEN = os.environ['GITHUB_TOKEN']
+
+        if GITHUB_TOKEN is None:
+            print('Github token not found')
+            return get_last_commit_date_by_terminal_log(file_path)
+        
+        headers = {
+            'X-GitHub-Api-Version': '2022-11-28',
+            'Accept': 'application/vnd.github+json',
+            'Authorization': f'Bearer {GITHUB_TOKEN}'
+        }
+
+        url = f'https://api.github.com/repos/OWASP/owasp-mastg/commits?path={file_path}'
```

Comment:
It was not working on local without this. It tried to request the local path to the API.
```suggestion
        path = file_path.split('owasp-mastg/')[1]
        url = f'https://api.github.com/repos/OWASP/owasp-mastg/commits?path={path}'
```